.infrastructure/98_variables.tf

```diff
@@ -63,4 +63,10 @@ variable "dns_domain_name" {
   description = "DNS domain for the b2b portals"
   type        = map(any)
   default     = null
+}
+
+variable "dns_delegate_records" {
+  type        = map(any)
+  description = "DNS delegate records"
+  default     = {}
```

Comment:
why do you need this variable? why is this variable needed in this activity?

Reply:
variable removed with last commit